skfda/exploratory/visualization/_utils.py

```diff
@@ -160,8 +166,13 @@ def _set_figure_layout(fig=None, axes=None,
     return fig, axes
 
 
-def _set_figure_layout_for_fdata(fdata, fig=None, axes=None,
-                                 n_rows=None, n_cols=None):
+def _set_figure_layout_for_fdata(
+    fdata: FData,
+    fig: Optional[Figure] = None, 
+    axes: Optional[List[Axes]] = None,
```

Comment:
```suggestion
    axes: Optional[Sequence[Axes]] = None,
```

Reply:
i tried changing it but it gave me errors on the code and couldnt do it, thats why i didnt use sequence